confluent_kafka/src/confluent_kafka.c

```diff
@@ -857,6 +884,10 @@ void Handle_clear (Handle *h) {
 		Py_DECREF(h->error_cb);
 	}
 
+	if (h->stats_cb) {
```

Comment:
style: remove braces for single-line clauses (you can remove the ones for error_cb above too, shouldnt be there)